### DISCUSSION TITLE
Attach metadata to an order

### DISCUSSION THREAD

**Discussion by barchard** [duncanmcclean/simple-commerce#999] [Feature Requests]
### Description

I’d like to tie a license to a specific domain then verify this domain and license combination matches via the API. Can SC be extended to include this? 

Thanks

**Comment by duncanmcclean** [comment]
Yes, I'd happily accept a PR for this.

**Comment by duncanmcclean** [comment]
The license validation endpoint will potentially be removed in the [next version of Simple Commerce](https://duncanmcclean.com/future-of-simple-commerce) as I think you're the only person using it - the download link will stay though.

This is probably something you can do yourself with line item metadata & a custom controller handling license validation.
